appveyor.yml

```diff
@@ -43,13 +49,10 @@ environment:
   TINYTDS_UNIT_AZURE_PASS:
     secure: fYKSKV4v+36OFQp2nZdX4DfUpgmy5cm0wuR73cgdmEk=
   matrix:
-    - ruby_version: "23-x64"
-    - ruby_version: "23"
-    - ruby_version: "24-x64"
     - ruby_version: "24"
     - ruby_version: "25-x64"
-    - ruby_version: "25"
-    - ruby_version: "26-x64"
     - ruby_version: "26"
+    - ruby_version: "head-x64"
```

Comment:
Some of the changes to this matrix list are surprising:

- why remove `24-x64`, `25`, and `26-x64`? If it's for overall speed concerns, I'd think that the ongoing verification is preferable to saving a few minutes per PR
- any limit against adding ruby 2.7?
- personally, I don't think testing against head builds is necessary, but I'm not opposed to it

Reply:
I removed some Ruby versions because one CI run takes almost two hours otherwise and testing all Ruby versions on add architectures has little value. Errors that affect only a single Ruby version on a single architecture are very rare.

Ruby-2.7 is not yet available in appveyor.

Testing on Ruby head is good to get early hints about incompatibilities or deprecations.

Reply:
> CI run takes almost two hours

Yeouch, I didn't know it was that bad.... I'm on board with slimming down the list then. Also this & the fact that it's so slow is making me want to revisit our overall CI config (later....)